wiki/Tournaments/TWC/2015/de.md

```diff
@@ -2,155 +2,526 @@
-6. Basierend auf [dieser Grafik](https://puu.sh/bUq5V/f1066103b0.png) teilen sich die Etappen wie folgt auf:
-   | Etappe | Match ID |
+
+3. Rankings für jede Gruppe werden durch eine Sortierung von den Ergebnissen anhand der Performance jedes Teams anhand der folgenden Prioritäten ermittelt:
+   1. Meiste gewonnene Partien.
+   2. Hat eine größere `{(Anzahl an gewonnenen Maps) | (Anzahl an verlorenen Maps)}`.
+   3. Meiste gewonnene Maps.
+   4. Hat eine größere `∑{(gesamte Ergebnisdifferenz) / (maximales Ergebnis)}`.
+   5. Gewinner des Rückspiels.
+
+4. Die besten 4 Teams in der Gruppe gehen weiter in die Knock-Out-Etappen.
+
+5. Die folgenden Etappen sind Doppeleliminierungsetappen. Dies bedeutet, dass der Gewinner in zur nächsten Etappe fortfährt und das andere Verliererteam in die Verliererklasse versetzt wird.
+
+6. Basierend auf [dieser Grafik](/wiki/shared/stages-visual.png) teilen sich die Etappen wie folgt auf:
+
+   | Etappe | Match-ID |
    | :-- | :-- |
    | Achtelfinale | A, B, C, D, E, F, G, H |
    | Viertelfinale | I, J, K, L & R, S, T, U |
    | Halbfinale | M, N & V, W, X, Y, Z,AA |
-   | Finale - Woche 1 | O & AB, AC, AD, AE |
-   | Finale - Woche 2 | P, Q |
-7. **Gewinnbedingungen:**
-   - In der Gruppenphase benötigen Sie 4 gewonnene Maps, um die Partie zu gewinnen. (Best-of-7)
-   - Im Achtel- und Viertelfinale benötigen Sie 5 gewonnene Maps, um die Partie zu gewinnen. (Best-of-9)
-   - Im Halbfinale und im Finale benötigen 6 gewonnene Maps, um die Partie zu gewinnen. (Best-of-11)
+   | Finalwoche 1 | O & AB, AC, AD, AE |
+   | Finalwoche 2 | P, Q |
+
+7. **Bedingungen, um zu gewinnen:**
+   - In der Gruppenphase musst du 4 Maps gewinnen, um die Partie zu gewinnen. (Best-of-7)
+   - Im Achtel- und Viertelfinale musst du 5 Maps gewinnen, um die Partie zu gewinnen. (Best-of-9)
+   - Im Halbfinale und im Finale musst du 6 Maps gewinnen, um die Partie zu gewinnen. (Best-of-11)
 
 ### Partieanweisungen
 
-1. Ein Schiedsrichter wird 15 Minuten im Vorhinein ein Multiplayer Raum erstellen. Die Spieler müssen sich während diesem Zeitraum versammeln.
-   1. Raumeinstellungen sind Taiko, Team-Vs., Win Condition: 'Score'. Raumname muss "TWC 2014: TeamBlue vs TeamRed" lauten.
-   2. Das Team, welches zuerst im Raumnamen genannt wurde, wird das blaue Team, das andere Team dementsprechend dann das rote Team.
-2. Den Spielern steht es zu 2 Beatmaps für die "Warm-Up" Phase zu spielen. Beatmaps mit fragwürdigem Inhalt sind verboten.
-3. Jeder Kapitän kann 2 Beatmaps aus dem Pool wählen, die nicht gespielt werden sollen. Dies gilt für beide Teams und die komplette Partie hindurch.
-   1. Beide Vetos müssen benutzt werden.
+1. Ein Schiedsrichter wird 15 Minuten vor Beginn einen Mehrspielerraum erstellen. Die Spieler müssen sich in dieser Zeit versammeln.
+   - Raumeinstellungen sind Taiko, Team-Vs., Siegbedingung : 'Score'. Raumname muss "TWC 2014: TeamBlue vs TeamRed" lauten.
```

Comment:
Maybe the extraneous space can be removed. But otherwise this article is good to go.
```suggestion
   - Raumeinstellungen sind Taiko, Team-Vs., Siegbedingung: 'Score'. Raumname muss "TWC 2014: TeamBlue vs TeamRed" lauten.
```